jenkins-pipelines/longevity-1tb-7days-azure.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
```

Comment:
need to change file name to reflect this new job being a 5 days long, and not 7 (i know you copied from another job that had that in name, but no one bothered yet to change it to 5 days, as it requires changing multiple Jenkins jobs)